### PR TITLE
cpu compliance

### PR DESCRIPTION
**Pull request by rvandeghen**
The code will work if cpu only

### CODE REVIEW

**Comment by rvandeghen** [comment]
Can you check if everuthing is ok for coco dataset a i only use custom dataset even though i have made the change in the coco codes

File: train.py [hunk 1]
```diff
@@ -118,8 +122,11 @@ def main(args=None):
             try:
                 optimizer.zero_grad()
 
-                classification_loss, regression_loss = retinanet([data['img'].cuda().float(), data['annot']])
-
+                if torch.cuda.is_available():
+                    classification_loss, regression_loss = model([data['img'].cuda().float(), data['annot']])
```
Comment (yhenon) [inline train.py]: are you sure this is right? why the change from `retinanet` to `model` ?

Comment (rvandeghen) [reply train.py]: It was a typo and I have changed it.

File: visualize.py [hunk 2]
```diff
@@ -50,7 +50,13 @@ def main(args=None):
 	use_gpu = True
 
 	if use_gpu:
-		retinanet = retinanet.cuda()
+        if torch.cuda.is_available():
```
Comment (yhenon) [inline visualize.py]: It appears you used spaces instead of tabs here

File: train.py (hunk 1 above)
Comment (yhenon) [reply train.py]: Thanks!

File: visualize.py (hunk 2 above)
Comment (rvandeghen) [reply visualize.py]: Changed it. Somehow i don't know why it was bad indented

**Comment by yhenon** [comment]
Thanks a lot for the PR @rvandeghen  !

**Comment by rvandeghen** [comment]
@yhenon No problem ! Actually I have made some other changes (add tensorboard, evaluation after each epoch,...) so if you are interested i can show you those changes also (private repo because of my Master thesis)
